docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -87,68 +151,196 @@ Users should keep the :ref:`storm-op-concepts` in mind when writing automation.
 Triggers and Cron
 -----------------
 
-Triggers and cron are similar in terms of permissions and management. Both triggers and cron:
-
-- **Support permissions.** Synapse uses permissions to determine who can create, modify, and delete triggers and cron jobs. Triggers and cron jobs execute **with the permissions of the user who creates them.** (This is fairly intuitive for cron, but may be less so in the case of triggers.) Conversely, a trigger or cron job can only perform actions that their creator has permissions to perform.
+Triggers and cron are similar in terms of how they are implemented and managed.
 
-- **Are introspectable.** Triggers and cron jobs are created as run-time nodes ("runt nodes") and can be lifted, filtered, and pivoted across just like other elements of the Synapse data model (see the :ref:`storm-ref-model-introspect` guide for details).
-
-- **Are view-specific.** Synapse allows the optional segregation of data in a Cortex into multiple layers (:ref:`gloss-layer`) that can be "stacked" to provide a unified :ref:`gloss-view` of data to users. Triggers and cron jobs are specific to a view. This means that they are carried over (for example) if a user forks (:ref:`gloss-fork`) a view. See the :ref:`storm-layer` and :ref:`storm-view` commands in the :ref:`storm-ref-cmd` guide for additional detail on working with views and layers.
-
-.. NOTE::
-  A simple Synapse implementation consists of a single Cortex with a single layer and a single view.
+- **Permissions.** Synapse uses permissions to determine who can create, modify, and delete triggers
+  and cron jobs. These permissions must be explicitly granted to users and/or roles.
+  
+- **Execution.** Both triggers and cron jobs execute with the permissions **of the user who creates them.**
+  A trigger or cron job can only perform actions that their creator has permissions to perform.
+  
+- **Introspection.** Triggers and cron jobs are created as run-time nodes ("runt nodes") in Synapse
```

Comment:
I might be being too picky here but it seems like we usually use "runtime" vs "run-time"